grasping_param_dict:  
  start: [1.287058373, 0.537414654849, 0.514067217975, -2.064004335, -1.07455044206, -0.71659260978, -0.990427049319]

  detector:
    row_2: [0.415935176792, -0.0417128288873, 0.353273315368, -1.56299992213, 2.74809383311, -1.47106497396, -0.37188408906]
    row_3: [0.567840996983, 0.284332294521, 0.660267037911, -1.85781574602, 2.85448457154, -1.27037631002, -0.61478154044]
    row_4: [0.567840996983, 0.284332294521, 0.660267037911, -1.85781574602, 2.85448457154, -1.27037631002, -0.61478154044]